src/System.CodeDom/tests/CodeCollections/CodeAttributeArgumentCollectionTests.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using Xunit;
+
+namespace System.CodeDom.Tests
+{
+	public class CodeAttributeArgumentCollectionTests
+	{
+		[Fact]
+		public void Ctor_IsEmpty()
+		{
+			var collection = new CodeAttributeArgumentCollection();
+			Assert.Equal(0, collection.Count);
+		}
+
+		public static IEnumerable<object[]> AddRange_TestData()
+		{
+			yield return new object[] { new CodeAttributeArgument[0] };
+			yield return new object[] { new CodeAttributeArgument[] { new CodeAttributeArgument() } };
```

Comment:
Since this is about adding a range, let's have at least three cases: 0, 1, and more than 1.  (If you've added similar tests later in the PR, same feedback applies).